docs/src/pages/components/pagination/examples/PaginationHasMorePagesExample.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { Pagination } from '@aws-amplify/ui-react';
+
+export const PaginationHasMorePagesExample = () => {
+  const [pageTokens, setPageTokens] = React.useState([
+    {
+      res: 'dog',
+      nextToken: 'page2',
+    },
+  ]);
+  const [currPageIndex, setCurrPageIndex] = React.useState(1);
```

Comment:
nit: prefer full words for readability
```suggestion
  const [currentPageIndex, setCurrentPageIndex] = React.useState(1);
```